site/src/css/components/_Carousel.scss

```diff
@@ -0,0 +1,158 @@
+/* Carousel CSS */
+
+@import './modules/CarouselSlide';
+
+.carousel-container {
+    width: 100%;
+    height: 550px;
+    display: flex;
+    flex-direction: column;
+    margin: 2em 0;
+
+    @include mobile-view {
+        height: 300px;
+        margin: 0;
+    }
+    @include ultra-slim-mobile-view {
+        height: 240px;
+    }
+
+    .decoration {
+        height: 50px;
+        width: 100%;
+        display: flex;
+
+        @include mobile-view {
+            display: none;
+        }
+    }
+    
+    .top {
+        margin-bottom: -1px;
+
+        svg {
+            transform: scaleY(-1);
+            width: 200px;
+            margin-left: 70%;
+        }
+    }
+    
+    .bottom {
+        margin-top: -1px;
+
+        svg {
+            width: 200px;
+            margin-right: 70%;
+            margin-left: auto;
+        }
+    }
+
+    .content {
+        width: 100%;
+        height: 450px;
+        background-color: var(--carousel-color);
+        overflow: hidden;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+
+        @include mobile-view {
+            height: 100%;
+        }
+
+        .carousel-header {
+            width: auto;
+            height: 50px;
+            color: var(--ifm-font-color-base);
+            font-family: 'Overpass Bold';
+            font-size: 32px;
```

Comment:
I believe its best practice to set font in `em` so that users who have changed default font size get the size relative to their default. 

We may need to address this in other places? Im sure a lot of the old original css i wrote does not follow best practices by any means

Reply:
`em` is good or you could use `rem` and make it all scalable from the root font size instead of the direct parent element. But yeah, since we're redoing CSS anyway we should remove the absolutes